ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -725,18 +725,13 @@ export const getMhrTransferHomeOwners = (state: StateIF): MhrRegistrationHomeOwn
   return owners
 }
 
-export const getMhrTransferCurrentHomeOwners = (state: StateIF): MhrRegistrationHomeOwnerGroupIF[] => {
-  const owners = []
+export const getMhrTransferCurrentHomeOwners = (state: StateIF): any => {
```

Comment:
Could we set a _type_ for the return value?

Reply:
We can for sure, will update

Reply:
Updated